unittests/test_pipeline.py

```diff
@@ -791,3 +791,51 @@ def extract_perf(patt, tag):
         self.assertIn('v1', log_output)
         self.assertIn('v2', log_output)
         self.assertIn('v3', log_output)
+
+
+class TestContainerPlatformsTest(TestRegressionTest):
+    def setup_test(self, platform, image):
+        self.setup_remote_execution()
+        test = self.loader.load_from_file(
+            'unittests/resources/checks_unlisted/containers.py')[0]
+        test.valid_prog_environs = [self.prgenv.name]
+        test.valid_systems = ['*']
+        test.container_platform = platform
+        test.container_platform.image = image
+        test.container_platform.commands = [
+            'pwd', 'ls', 'cat /etc/os-release']
+        test.container_platform.workdir = '/workdir'
+        test.sanity_patterns = sn.all([
+            sn.assert_found(r'^' + test.container_platform.workdir,
+                            test.stdout),
+            sn.assert_found(r'^container_test.txt', test.stdout),
+            sn.assert_found(r'18.04.3 LTS \(Bionic Beaver\)', test.stdout),
+        ])
+        return test
+
+    def run_test(self, test):
+        try:
+            self._run_test(test)
+        except PipelineError:
+            self.skipTest('no configuration found for container platform: %s' %
+                          type(test.container_platform).__name__)
```

Comment:
No, you can't skip the test like that here. The fact that a `PipelineError` was thrown does not mean that it is valid to skip the test. It might be a bug. You should check individually here that a test may not be run and only then skip the test.